src/applications/ivc-champva/10-10D/sass/10-10D.scss

```diff
@@ -11,3 +11,8 @@
 .upload-button-label.vads-u-display--inline-block {
   margin-top: initial;
 }
+
+va-accordion-item[data-chapter="uploadFiles"] {
+  display: none;
+  visibility: hidden;
+}
```

Comment:
Hides the `uploadFiles` chapter from the review accordion. Have to do it here because the two custom pages in that chapter are set not to show on review, but the chapter itself still does for some reason.